ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,392 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 7854
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.